packages/better-auth/src/plugins/app-invite/adapter.ts

```diff
@@ -0,0 +1,93 @@
+import type { AppInviteOptions } from ".";
+import type { AuthContext, User } from "../../types";
+import { getDate } from "../../utils/date";
+import { type AppInvitationInput, type AppInvitation } from "./schema";
+
+export const getAppInviteAdapter = (
+	context: AuthContext,
+	options?: AppInviteOptions,
+) => {
+	const adapter = context.adapter;
+	return {
+		createInvitation: async ({
+			invitation,
+			user,
+		}: {
+			invitation: {
+				email?: string;
+				domainWhitelist?: string;
+			};
+			user: User;
+		}) => {
+			const defaultExpiration = 1000 * 60 * 60 * 48;
+			const expiresAt = getDate(
+				options?.invitationExpiresIn || defaultExpiration,
+			);
+			const invite = await adapter.create<AppInvitationInput, AppInvitation>({
+				model: "appInvitation",
+				data: {
+					email: invitation.email,
+					status: "pending",
+					expiresAt,
+					inviterId: user.id,
+					domainWhitelist: !invitation.email
+						? invitation.domainWhitelist
+						: undefined,
+				},
+			});
+
+			return invite;
+		},
+		findInvitationById: async (id: string) => {
```

Comment:
I noticed you added a way to get one invite but no way to get all of the invitations (can be useful for an admin dashboard)

```suggestion
		findManyInvitations: async (input: { where?: Where[] }) => {
			const invitation = await adapter.findMany<AppInvitation>({
				model: "appInvitation",
				where: input.where,
			});
			return invitation;
		},
		findInvitationById: async (id: string) => {
```